package.json

```diff
@@ -11,7 +11,9 @@
     "packageQuick": "npm run build && electron-builder --dir",
     "format": "prettier --write src/**/*",
     "CIbuild": "npm run build && electron-builder --linux zip && electron-builder --windows zip && electron-builder --macos zip",
-    "postinstall": "husky install"
+    "postinstall": "husky install",
+    "precommit-fix": "husky uninstall && node fixwarn.js",
+    "precommit-fix:format": "husky uninstall && npm run format"
```

Comment:
...what's the point of uninstalling Husky's hooks? They automatically format the files before you commit.

Reply:
Husky has trouble working on Windows (there's even official documentation on some workarounds) so incase nothing works, this would be a valid option

Reply:
It is not a valid option because this prevents Linux users from using the hooks as well. Besides, I added the hooks on a Windows machine and to this day they work fine for me.

Reply:
It doesn't break Husky, it's just a command for people like me who have issues on Windows. And as for your situation, it's not that it breaks all the time, but the fact that it's possible, notably after building the app for testing. (My situation)